src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -163,124 +163,20 @@
-						@click="createAndJoinConversation(item)">
-						<template #icon>
-							<AvatarWrapper v-bind="iconData(item)" />
-						</template>
-						<template v-if="!isCompact" #subname>
-							{{ t('spreed', 'New private conversation') }}
-						</template>
-					</NcListItem>
-				</template>
-
-				<!-- Search results: new conversations -->
-				<template v-if="canStartConversations">
-					<!-- New conversations: Groups -->
-					<template v-if="searchResultsGroups.length !== 0">
-						<NcAppNavigationCaption :name="t('spreed', 'Groups')" />
-						<NcListItem v-for="item of searchResultsGroups"
-							:key="`group_${item.id}`"
-							:data-nav-id="`group_${item.id}`"
-							:name="item.label"
-							:compact="isCompact"
-							@click="createAndJoinConversation(item)">
-							<template #icon>
-								<ConversationIcon :item="iconData(item)" :size="isCompact ? AVATAR.SIZE.COMPACT: AVATAR.SIZE.DEFAULT" />
-							</template>
-							<template v-if="!isCompact" #subname>
-								{{ t('spreed', 'New group conversation') }}
-							</template>
-						</NcListItem>
-					</template>
-
-					<!-- New conversations: Circles -->
-					<template v-if="searchResultsCircles.length !== 0">
-						<NcAppNavigationCaption :name="t('spreed', 'Teams')" />
-						<NcListItem v-for="item of searchResultsCircles"
-							:key="`circle_${item.id}`"
-							:data-nav-id="`circle_${item.id}`"
-							:name="item.label"
-							:compact="isCompact"
-							@click="createAndJoinConversation(item)">
-							<template #icon>
-								<ConversationIcon :item="iconData(item)" :size="isCompact ? AVATAR.SIZE.COMPACT: AVATAR.SIZE.DEFAULT" />
-							</template>
-							<template v-if="!isCompact" #subname>
-								{{ t('spreed', 'New group conversation') }}
-							</template>
-						</NcListItem>
-					</template>
-
-					<!-- New conversations: Federated users -->
-					<template v-if="searchResultsFederated.length !== 0">
-						<NcAppNavigationCaption :name="t('spreed', 'Federated users')" />
-						<NcListItem v-for="item of searchResultsFederated"
-							:key="`federated_${item.id}`"
-							:data-nav-id="`federated_${item.id}`"
-							:name="item.label"
-							:compact="isCompact"
-							@click="createAndJoinConversation(item)">
-							<template #icon>
-								<AvatarWrapper v-bind="iconData(item)" />
-							</template>
-							<template v-if="!isCompact" #subname>
-								{{ t('spreed', 'New group conversation') }}
-							</template>
-						</NcListItem>
-					</template>
-				</template>
-
-				<!-- Search results: no results (yet) -->
-				<template v-if="sourcesWithoutResults">
-					<NcAppNavigationCaption :name="sourcesWithoutResultsList" />
-					<Hint :hint="t('spreed', 'No search results')" />
-				</template>
-				<Hint v-else-if="contactsLoading" :hint="t('spreed', 'Loading …')" />
-			</ul>
+			<SearchConversationsResults v-else
+				ref="searchResults"
+				class="scroller"
+				:search-text="searchText"
+				:is-compact="isCompact"
```

Comment:
Like in another virtual component
```suggestion
				:compact="isCompact"
```